apps/ledger-live-mobile/src/families/elrond/accountActions.tsx

```diff
@@ -26,16 +26,22 @@ export type getActionsReturnType = ActionButtonEvent[] | null | undefined;
 const getActions = (props: getActionsType): getActionsReturnType => {
   const { account } = props;
 
-  const balance = denominate({ input: String(account.spendableBalance) });
-  const delegationEnabled = new BigNumber(balance).gt(1);
+  const balance = denominate({
+    input: String(account.spendableBalance),
+    showLastNonZeroDecimal: true,
+  });
+  const delegationEnabled = new BigNumber(balance).gte(1);
 
   /*
    * Get a list of all the providers, randomize, and also the screen, conditionally, based on existing amount of delegations.
    */
 
-  const validators = randomizeProviders(account.elrondResources.providers);
+  const validators = randomizeProviders(
+    account.elrondResources ? account.elrondResources.providers : [],
+  );
+
   const screen =
-    account.elrondResources.delegations.length === 0
+    account.elrondResources && account.elrondResources.delegations.length === 0
```

Comment:
Doesn't seem mandatory as `account` is of type `ElrondAccount`.

Reply:
Well, there is an use case where it's coming back as TokenAccount, instead of ElrondAccount, which doesn't have the `elrondResources` key. And this checks for that in order to prevent an application crash.